dev/packages/alpha/cisco/0.0.3/dataset/ios/agent/stream/stream.yml.hbs

```diff
@@ -0,0 +1,236 @@
+paths:
+{{#each paths as |path i|}}
+  - {{path}}
+{{/each}}
+exclude_files: [".gz$"]
+tags: {{this.tags}}
```

Comment:
Further above you just us `{{tags}}`. Do both work?

Reply:
both work, but I will clean this up.